examples/sample-input-byo.tfvars

```diff
@@ -45,6 +45,10 @@ default_nodepool_custom_data            = ""
 ## General
 efs_performance_mode                    = "maxIO"
 storage_type                            = "standard"
+# To enable EBS encyption - set to 'true'. Variable is set to 'false' by default.
+enable_ebs_encryption = false
+# To enable EFS encyption - set to 'true'. Variable is set to 'false' by default.
+enable_efs_encryption = false
```

Comment:
Just wondering why we included these elements in the sample files if the values are only applying the default value of `false`? Does the CONFIG-VARS.md file not cover the addition of these flags? This comment holds for all sample files where the changes were made.

Reply:
CONFIG-VARS.md does not cover these flags. Should I remove them from the tfvars files and cover them in the CONFIG-VARS.md?

Reply:
Added EBS and EFS elements to CONFIG-VARS.md. Updates are up now.